### ISSUE TITLE
Use of Cash Account as address in signature verification dialog?

### ISSUE THREAD

**Issue by molecular** [Electron-Cash/Electron-Cash#2168]
I just had the following use case:

Someone sent me a signed message and I wanted to verify the signature. 

He signed using a Cash Account address.

Wouldn't it be nifty if I could just use his Cash Account in "Address" field of Signature Verification Dialog?

I suspect there may be some caveats?

**Comment by cculianu** [comment]
Yeah that's possible. No real caveats I guess it would just have to resolve it and if it fails to verify it, complain.  Could be done.  Will leave issue open to remember to get to it.
